gateway/src/resty/ctx.lua

```diff
@@ -19,11 +19,17 @@ local tonumber = tonumber
 local _M = {
 }
 
+local function __ngx_req()
+  return getfenv(0).__ngx_req
+end
+
+local get_request = base.get_request or __ngx_req
```

Comment:
Do we need the `or` in 1.15.8.1 or just to keep it working in 1.13.x ?

Reply:
The or is for 1.13. `base.get_request` first appears in `1.15`. It could be cleaned up later.